src/screens/createAccount/index.tsx

```diff
@@ -84,13 +109,48 @@ const CreateAccount = () => {
         control={control}
         name="repeatPassword"
         placeholder="Repeat password"
-        label="Repeat password"
         onFocus={() => onFocusInput('repeatPassword')}
         error={errors.repeatPassword?.message}
         type="password"
       />
+      <Input
+        control={control}
+        name="skill"
+        placeholder="Skill Score"
+        onFocus={() => onFocusInput('skill')}
+        error={errors.skill?.message}
+        rightElement={
+          <View m={2}>
+            <Popover trigger={props => <InfoIcon {...props} />}>
```

Comment:
We could create a shared component for the Popover and move all this to another file

Reply:
Done!